var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -28,6 +28,7 @@ class PyNumpy(PythonPackage):
     version("2.0.2", sha256="883c987dee1880e2a864ab0dc9892292582510604156762362d9326444636e78")
     version("2.0.1", sha256="485b87235796410c3519a699cfe1faab097e509e90ebb05dcd098db2ae87e7b3")
     version("2.0.0", sha256="cf5d1c9e6837f8af9f92b6bd3e86d513cdc11f60fd62185cc49ec7d1aba34864")
+    version("1.26.5", sha256="4e9dc074d7b0c5d00b82c3e88d1f1182f580ee0db9ea3b823330f4c2f5e1d79e")
```

Comment:
I am unable to confirm this sha256. I get `Error: Could not find any remote versions for py-numpy`

Reply:
Seems like this version exists on GitHub but not on PyPI.

Reply:
It's indeed not a proper release. The release manager just tagged a bunch of commits with `v1.26.5` for some reason to make , that probably shouldn't have been done. There is nothing critical in commits beyond `1.26.4`, so I'd personally prefer for distros not to provide a `1.26.5`.

Reply:
OK, I removed 1.26.5.